### PR TITLE
virtio_fs:Update virtio-fs configuration steps on S390

### PR DESCRIPTION
**Pull request by zhenyzha**
Update virtio-fs configuration steps on S390

ID:2008086 
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
Test All Pass
@PaulYuuu @fbq815 Could you pls help review, when you are free. thanks. 

```
 (02/98) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fsd_check_info.
s390-virtio: PASS (1.72 s)  
 (03/98) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_share_data.w
ith_cache.auto.s390-virtio: PASS (40.34 s)
......
 (25/98) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_share_data.w
ith_cache.none.xattr_on.flock_off.lock_posix_off.s390-virtio: PASS (42.75 s)
 (26/98) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_share_data.w
ith_cache.none.xattr_on.flock_off.lock_posix_on.s390-virtio: PASS (41.61 s)

 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_share_data.with_multi_fs_sources.with_cache.auto.s390-virtio: PASS (52.45 s)

 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_share_data_multi_backend.with_cache.auto.with_nfs_source.s390-virtio: PASS (29.43 s)

 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_dac_read_search.with_cache.auto.s390-virtio: PASS (26.02 s)

 (1/1) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_readonly.s390-virtio: PASS (24.43 s)
```

File: qemu/tests/cfg/virtio_fs_multi_vms.cfg [hunk 1]
```diff
@@ -1,11 +1,10 @@
 - virtio_fs_multi_vms:
+    no s390 s390x
```
Comment (PaulYuuu) [inline qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: Can you explain this change?

File: qemu/tests/cfg/virtio_fs_set_capability.cfg [hunk 2]
```diff
@@ -60,6 +66,8 @@
             cmd_create_fs_source = 'mkdir -p /tmp/virtio_fs1_test'
             cmd_run_virtiofsd = '/usr/libexec/virtiofsd --socket-path=%s'
             cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=2048 oflag=direct'
+            s390, s390x:
+                cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=200 oflag=direct'
```
Comment (PaulYuuu) [inline qemu/tests/cfg/virtio_fs_set_capability.cfg:70]: OK, this is more like a host resource limitation, but it's okay here.

File: qemu/tests/cfg/virtio_fs_multi_vms.cfg (hunk 1 above)
Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: This's also the reason for host resource limitations. A VM image is probably 5G.
Unfortunately on S390 this often leads to failure.

Comment (PaulYuuu) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: So this can be regarded as not a generic problem, we can only let it out of our test scope.

Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg:2]: ok, update later.

Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg]: It seems that reducing the number of VMS images allows this case to pass in few s390 host environments.

Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_multi_vms.cfg]:
```
 (1/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_multi_vms.with_multi_fs_sources.s390-virtio: PASS (38.81 s)
 (2/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_multi_vms.share_fs_source.s390-virtio: PASS (37.38 s)
```

**Comment by zhenyzha** [comment]
@PaulYuuu @fbq815 Could you pls help review again when you are free. thanks.

**Review by PaulYuuu — APPROVED** [review]

**Review by fbq815 — APPROVED** [review]

**Comment by zhenyzha** [comment]
git am 3024.patch
Applying: virtio_fs:Update virtio-fs configuration steps on S390

```
 (1/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_multi_vms.with_multi_fs_sources.s390-virtio: PASS (38.71 s)
 (2/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.s390x.io-github-autotest-qemu.virtio_fs_multi_vms.share_fs_source.s390-virtio: PASS (37.11 s)
```

**Pull request merged**